src/System.Memory/src/System/SpanHelpers.T.cs

```diff
@@ -120,8 +121,9 @@ public static unsafe int IndexOf<T>(ref T searchSpace, T value, int length)
         }
 
         public static bool SequenceEqual<T>(ref T first, ref T second, int length)
-            where T : struct, IEquatable<T>
+            where T : IEquatable<T>
         {
+            //if (typeof(T) == typeof(byte)) return SequenceEqual(ref Unsafe.As<T, byte>(ref first), ref Unsafe.As<T, byte>(ref second), length);
```

Comment:
Why is this commented out?

Reply:
It's a typo from when I was doing before/after measurements. Will fix.